### PR TITLE
[CMAKE] Activate graph tests

### PR DESCRIPTION
**Pull request by markuspf**
### Scope & Purpose

Kinda wonder whether we should be running these.

### CODE REVIEW

File: tests/CMakeLists.txt [hunk 1]
```diff
@@ -418,6 +418,7 @@ add_executable(
   ../cmake/activeCodePage.manifest
   ${ARANGODB_TESTS_SOURCES}
   ${ARANGODB_REPLICATION2_TEST_SOURCES}
+  ${ARANGODB_GRAPH_TEST_SOURCES}
```
Comment (mchacki) [inline tests/CMakeLists.txt:421]: I cannot targetly comment, as the lines below that are not touched do not allow it.

However can you add this line also to line 442?

```
if (USE_SEPARATE_GRAPH_TESTS_BINARY)
  add_executable(
          arangodbtests_graph
          ${ARANGODB_TESTS_COMMON_SOURCES}
          main.cpp
  )
```

=>

```
if (USE_SEPARATE_GRAPH_TESTS_BINARY)
  add_executable(
          arangodbtests_graph
          ${ARANGODB_TESTS_COMMON_SOURCES}
          ${ARANGODB_GRAPH_TEST_SOURCES}
          main.cpp
  )
```

Otherweise the "graph_only" test binary is just empty

**Review by mchacki — CHANGES_REQUESTED** [review]
Please add it into another position:

The change itself is great.

File: tests/CMakeLists.txt (hunk 1 above)
Comment (markuspf) [reply tests/CMakeLists.txt:421]: Done. As the tests got broken in the meantime, I'll now look at how to fix them.

**Review by mchacki — APPROVED** [review]
LGTM

**Pull request merged**